rust/cubesqlplanner/cubesqlplanner/src/cube_bridge/base_query_options.rs

```diff
@@ -59,6 +59,8 @@ pub struct BaseQueryOptionsStatic {
     pub row_limit: Option<String>,
     pub offset: Option<String>,
     pub ungrouped: Option<bool>,
+    #[serde(rename = "exportAnnotatedSql")]
```

Comment:
Btw, instead of tagging every property, you can tag the struct with `#[serde(rename_all = "camelCase")]` and it will do everything you need :) but it's just one line  → code looks cleaner and more compact.